prboom2/src/SDL/i_sound.c

```diff
@@ -227,6 +227,23 @@ static int addsfx(int sfxid, int channel, const unsigned char *data, size_t len)
   return channel;
 }
 
+static int getBufferSize(void)
```

Comment:
If you are taking the function from Chocolate Doom, why do you rename it and the variables it uses?

Reply:
I was going to rewrite it, but then realized that would be silly.